app/views/hyrax/admin/admin_sets/_form_visibility.html.erb

```diff
@@ -44,7 +44,7 @@
                   <h3><%= t('.visibility.title') %></h3>
                   <p><%= t('.visibility.description') %></p>
                   <%# List each option in a <div class='radio'> tag %>
-                  <%= f.collection_radio_buttons :visibility, f.object.visibility_options, :first, :last, item_wrapper_tag: :div, item_wrapper_class: 'radio' do |b| %>
+                  <%= f.collection_radio_buttons :visibility, f.object.visibility_options, :first, :last, item_wrapper_tag: :div, item_wrapper_class: 'radio', checked: f.object[:visibility] do |b| %>
```

Comment:
This was not behaving as default in this situation, and making it explicit solves the issue. I'm not sure why it wasn't finding it by default... perhaps because the permission template has so many levels and it wasn't finding the right items to check? The collection of radio buttons is based on f.object.visibility_options, but the checked item comes from f.object[:visibility]